src/interval.jl

```diff
@@ -171,6 +171,22 @@ function Base.last(interval::Interval{T,L,R}) where {T,L,R}
     return R !== Unbounded ? interval.last : nothing
 end
 
+function Base.minimum(interval::Interval{T,Closed,R}; precision=nothing) where {T,R}
```

Comment:
Just for consistency:
```suggestion
function Base.minimum(interval::Interval{T,Closed,R}; precision=eps(T)) where {T,R}
```

Reply:
Actually, I take this back. You can leave it as `nothing`. I thought Julia was smart enough to optimize this away. That isn't the case.
```julia
julia> f(x=nothing) = 1
f (generic function with 2 methods)

julia> g(x=eps(Int)) = 1
g (generic function with 2 methods)

julia> @code_native f()
	.section	__TEXT,__text,regular,pure_instructions
; ┌ @ REPL[5]:1 within `f'
	movl	$1, %eax
	retq
	nopw	%cs:(%rax,%rax)
; └

julia> @code_native g()
	.section	__TEXT,__text,regular,pure_instructions
; ┌ @ REPL[6]:1 within `g'
	pushq	%rax
	movabsq	$jl_system_image_data, %rax
	movq	%rax, (%rsp)
	movabsq	$jl_apply_generic, %rax
	movabsq	$jl_system_image_data, %rdi
	movq	%rsp, %rsi
	movl	$1, %edx
	callq	*%rax
	ud2
	nop
; └

julia> g()
ERROR: MethodError: no method matching eps(::Type{Int64})
Closest candidates are:
  eps(::Dates.Time) at /Users/omus/Development/Julia/1.3/usr/share/julia/stdlib/v1.3/Dates/src/types.jl:387
  eps(::Dates.Date) at /Users/omus/Development/Julia/1.3/usr/share/julia/stdlib/v1.3/Dates/src/types.jl:386
  eps(::Dates.DateTime) at /Users/omus/Development/Julia/1.3/usr/share/julia/stdlib/v1.3/Dates/src/types.jl:385
  ...
Stacktrace:
 [1] g() at ./REPL[6]:1
 [2] top-level scope at REPL[9]:1
```